tests/Fakes/FakeReflectionClass.php

```diff
@@ -30,12 +30,12 @@ public function withoutNamespace(): self
         return $this;
     }
 
-    public function getNamespaceName()
+    public function getNamespaceName(): string
     {
         return $this->withNamespace ?? parent::getNamespaceName();
```

Comment:
If `$this->withNamespace` is not nullable anymore, this should be changed to:

```suggestion
        return $this->withNamespace ?: parent::getNamespaceName();
```